__tests__/app.test.js

```diff
@@ -47,3 +47,27 @@ describe('GET /api/topics', () => {
             });
     });
 });
+describe('GET /api', () => {
+    test('200: should return an object containing the description for this endpoint', () => {
+        return request(app)
+            .get('/api')
+            .expect(200)
+            .then(({body}) => {
+                expect(typeof body.endpoints).toBe('object');
+                expect(Object.keys(body.endpoints['GET /api'])).toContain('description');
+                expect(typeof body.endpoints['GET /api'].description).toBe('string');
```

Comment:
This testing is completely valid, however, if you wanted to be even more thorough, you can simply require in the same JSON file in your test and assert it is sent as a response for this end point.  😊